### PR TITLE
added saakuru chain

### PR DESCRIPTION
**Pull request by aag-victor**

### CODE REVIEW

File: _data/chains/eip155-247253.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://saakuru.network",
+  "shortName": "OAS", 
+  "chainId": 247253,
+  "networkId": 247253,
+  "explorers": [
+    {
+      "name": "saakuru-explorer-testnet",
+      "url": "https://explorer-testnet.saakuru.network/",
```
Comment (ligi) [inline _data/chains/eip155-247253.json:19]:
```suggestion
      "url": "https://explorer-testnet.saakuru.network",
```

Comment (ligi) [inline _data/chains/eip155-247253.json:19]:
```suggestion
      "url": "https://explorer-testnet.saakuru.network",
```

File: _data/chains/eip155-7225878.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://saakuru.network",
+  "shortName": "OAS", 
+  "chainId": 7225878,
+  "networkId": 7225878,
+  "explorers": [
+    {
+      "name": "saakuru-explorer",
+      "url": "https://explorer.saakuru.network/",
```
Comment (ligi) [inline _data/chains/eip155-7225878.json:19]:
```suggestion
      "url": "https://explorer.saakuru.network",
```

**Review by ligi — APPROVED** [review]

**Review by ariel3112 — APPROVED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `OAS` more than once

**Comment by aag-victor** [comment]
short name changed. please check. thanks!

**Review by ligi — APPROVED** [review]

**Pull request merged**